### DISCUSSION TITLE
Validating portion(s) of a dynamic text using assertVisible

### DISCUSSION THREAD

**Discussion by rbenitezpagan** [mobile-dev-inc/Maestro#1660] [Q&A] [answered]
### Problem
I am having problems successfully validating a dynamic text (or portions of it).

### Question
Is it possible to validate a dynamic text if I split the validations?

## Input
`Lorem ipsum dolor sit amet, <<dynamic text>>, consectetur adipiscing elit.`

## Flow
```
- assertVisible: "Lorem ipsum dolor sit amet, "
# ignore <<dynamic text>>
- assertVisible: ", consectetur adipiscing elit."
```

### Outputs
`Assertion is false: "Lorem ipsum dolor sit amet, " is visible`
`Assertion is false: ", consectetur adipiscing elit." is visible`

**Comment by simon-gilmurray** [comment, marked as answer]
You're currently checking for a full string to match, and as that doesn't exist it's failing.

You can use regex to check partials, e.g.

```
- assertVisible: Lorem ipsum dolor sit amet.*
# ignore <<dynamic text>>
- assertVisible: .*consectetur adipiscing elit
```

**Reply by rbenitezpagan** [reply]
Thanks @simon-gilmurray ❤
